modules/account-lib/src/coin/hbar/transaction.ts

```diff
@@ -84,16 +85,37 @@ export class Transaction extends BaseTransaction {
    * Get the recipient account and the amount
    * transferred on this transaction
    *
-   * @returns {[string, string]} - First element is the recipient, second element is the amount
+   * @returns {{recipient: string; amount: string; tokenName?: string}} consisting
+   *  the recipient, the transfer amount, and the token name for token transfer
    */
-  private getTransferData(): [string, string] {
+  private getTransferData(): { recipient: string; amount: string; tokenName?: string } {
```

Comment:
You'll need to update this method to account for multi recipient transfers, as well as the `TxData` interface in `iface.ts`

Reply:
will move it to `iface.ts`, and TxData update during next PR